iabtcf-gvl-jackson/src/main/java/com/iabtcf/gvl/jackson/Vendor.java

```diff
@@ -0,0 +1,159 @@
+     * are just marked as deleted
+     *
+     * @return vendor id
+     */
+    public int getId() {
+        return id;
+    }
+
+    /**
+     * Name of the vendor
+     *
+     * @return vendor name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * List of Purposes for which the vendor is requesting consent
+     *
+     * @return A {@link List} of purpose ids that require consent
+     */
+    public List<Integer> getPurposes() {
+        return purposes;
+    }
+
+    /**
+     * List of Purposes for which the vendor requires to be transparently disclosed as their legitimate interest
+     *
+     * @return A {@link List} of purpose ids disclosed as legitimate interests
+     */
+    public List<Integer> getLegIntPurposes() {
+        return legIntPurposes;
+    }
+
+    /**
+     * List of purposes where the vendor is flexible regarding the legal basis; they will perform the processing
+     * based on consent or a legitimate interest. The 'default' is determined by which of the other two
+     * mutually-exclusive purpose fields is used to declare the purpose for the vendor
+     *
+     * @return A {@link List} of flexible purpose ids
+     */
+    public List<Integer> getFlexiblePurposes() {
+        return flexiblePurposes;
+    }
+
+    /**
+     * List of Special Purposes that the vendor transparently discloses as their legitimate interest that a user
+     * has no right to object
+     *
+     * @return A {@link List} of special purpose ids
+     */
+    public List<Integer> getSpecialPurposes() {
+        return specialPurposes;
+    }
+
+    /**
+     * List of Features the vendor uses across Purposes
+     *
+     * @return A {@link List} of features
+     */
+    public List<Integer> getFeatures() {
+        return features;
+    }
+
+    /**
+     * List of Special Features the vendor uses across Purposes
+     *
+     * @return A {@link List} of special features
+     */
+    public List<Integer> getSpecialFeatures() {
+        return specialFeatures;
+    }
+
+    /**
+     * GDPR/privacy policy page URL
+     *
+     * @return policy url string
+     */
+    public String getPolicyUrl() {
+        return policyUrl;
+    }
+
+    /**
+     * A date string representing when the vendor is deleted from the GVL
+     *
+     * @return date string
+     */
+    public String getDeletedDate() {
+        return deletedDate;
+    }
+
+    /**
+     * object specifying the vendor's http GET request length limit. It is optional.
+     * If a vendor entry does not include this attribute then the vendor has no
+     * overflow options and none can be inferred.
+     *
+     * @return A {@link com.iabtcf.gvl.Overflow} object
+     */
+    public Overflow getOverflow() {
+        return overflow;
+    }
+
+    /**
+     * A helper method to check if the vendor is deleted based on the current time
+     *
+     * @return true, if the vendor is deleted
+     */
+    public boolean isActive() {
```

Comment:
rename to isDeleted(), "active" is an undefined term in the specification unlike deleted